docker-compose.openSearch.yml

```diff
@@ -28,15 +28,40 @@ services:
       - "OPENSEARCH_HOSTS=${OPENSEARCH_HOSTS:-http://opensearch:9200}"
       - "ELASTICSEARCH_USERNAME=${ELASTICSEARCH_USERNAME:-admin}"
       - "ELASTICSEARCH_PASSWORD=${ELASTICSEARCH_PASSWORD:-admin}"
+      - 'SERVER_BASEPATH="/opensearch"'
+      - 'SERVER_REWRITEBASEPATH=true'
     volumes:
-      - "./config/kibana.yml:/usr/share/opensearch-dashboards/config/kibana.yml"
+      - "./opensearch.yml:/usr/share/opensearch-dashboards/config/opensearch.yml"
     networks:
       openimis-net:
         ipv4_address: 172.20.${IP_SUB:-20}.97
 
+  nginx:
+    build:
+      context: .
+      dockerfile: ${DOCKERFILE_NAME:-Dockerfile-nginx-}
+    ports:
+      - "80:80"
```

Comment:
On most recent develop port 80 is reserved by the frontend container. Just want to confirm that it's possible to run all containers at once without excessive reconfiguration.

Reply:
What about 443 port? If the certificate is provided for the frontend container will opensearch still work (like on test server)? Perhaps better solution would be to use nginx configuration for opensearch to update frontend container nginx, instead of creating separate container (for fully dockerized instance).

Reply:
It works in such combination:
![Uploading Peek 2023-07-04 14-06.gif…]()